pkg/virt-operator/resource/apply/delete_test.go

```diff
@@ -169,6 +175,76 @@ var _ = Describe("Deletion", func() {
 			Expect(needRemoved).To(BeEmpty())
 		})
 	})
+
+	Context("Node label deletion", func() {
+		var clientset *fake.Clientset
+		var kubevirtClient *kubecli.MockKubevirtClient
+		var ctrl *gomock.Controller
+
+		BeforeEach(func() {
+			ctrl = gomock.NewController(GinkgoT())
+			clientset = fake.NewSimpleClientset()
+			kubevirtClient = kubecli.NewMockKubevirtClient(ctrl)
+			kubevirtClient.EXPECT().CoreV1().Return(clientset.CoreV1()).AnyTimes()
+		})
+
+		AfterEach(func() {
+			ctrl.Finish()
+		})
+
+		It("should delete kubevirt labels from nodes", func() {
+			nodes := &k8sv1.NodeList{
+				Items: []k8sv1.Node{
```

Comment:
You need to adjust tests with `kubevirt.io/schedulable` label.
You can also add a node without it and check that it will be skipped.